jib-core/src/main/java/com/google/cloud/tools/jib/docker/DockerContextGenerator.java

```diff
@@ -84,7 +84,8 @@ static String joinAsJsonArray(List<String> items) {
   @VisibleForTesting
   static String makeExposeItems(List<String> exposedPorts) {
     return String.join(
-        "\n", exposedPorts.stream().map(port -> "EXPOSE " + port).collect(Collectors.toList()));
+        System.getProperty("line.separator"),
```

Comment:
That's interesting. I'm wondering why this would need this platform-specific separator when the template's lines are separated by just LF?

Also, was the error with Docker not accepting non CRLF on windows?

Reply:
I didn't look that deeply into it, the error just said that the sample and the generated docker file lengths differed by 1 on windows, so I assumed it was using "\r\n" as separators and was missing a "\r".

Reply:
Hmm that might mean the test should be fixed. I think the docker context generated should be platform independent so that a user generating on linux or windows would produce the same context for the same project.